README.md

```diff
@@ -26,13 +26,11 @@ You need:
 
 * A "normal" filesystem in which files stay put forever, *OR* Amazon S3, *OR* Microsoft Azure, *OR* Google Cloud Platform OR a willingness to write a backend for something else (look at `s3.js`, `azure.js` and `local.js` for examples; just supply an object with the same methods, you don't have to supply a factory function).
 
-* Patience, to wait for [Jimp](https://github.com/oliver-moran/jimp) to convert your images; or [Imagemagick](http://www.imagemagick.org/script/index.php), if you want much better speed and GIF support. You can also write a backend for something else (look at `imagemagick.js`, `imagecrunch.js`, and `jimp.js` for examples; just supply an object with the same methods, you don't have to supply a factory function).
+* Patience, to wait for [Sharp](https://www.npmjs.com/package/sharp) to convert your images; or optionally, [Imagemagick](http://www.imagemagick.org/script/index.php). You can also write a backend for something else (look at `sharp.js` or `imagemagick.js` for examples); just supply an object with the same methods, you don't have to supply a factory function).
```

Comment:
"Patience" is not required with Sharp, it's faster than Imagemagick. This language was designed to encourage use of Imagemagick instead of Jimp.

You could just write:

"Most modern macOS, Windows and Linux systems running Node.js >= 12.13.0 do not require any additional install or runtime dependencies. They will automatically use `sharp` which is extremely fast. Systems not meeting these qualifications can still use this module if `imagemagick` is installed on the system."

My requirements language there is borrowed from the sharp module documentation.

Reply:
Changed.